SFML/SFML
# Patch out include & pkgconfig files for external libraries

## Description

This is an alternative approach to #3341 or https://github.com/SFML/SFML/compare/master...exclude_from_all

Disabled header installing for freetype and patched it our for FLAC, Vorbis and Ogg.

This still install pkgconfig files, should we remove those as well?

## Tasks


## How to test this PR?

Build the install target and check that no unwanted third-party files is being installs.
